modules/core/map/js/farmOS.map.behaviors.input.js

```diff
@@ -1,11 +1,56 @@
 (function () {
   farmOS.map.behaviors.input = {
     attach: function (instance) {
+
+      // Get the data input form element.
+      var input = instance.map.getTargetElement().parentElement.querySelector('[data-map-geometry-field]')
+
+      // When features change in the edit layer, write WKT to the input form element.
       instance.editAttached && instance.editAttached.then(() => {
         instance.edit.wktOn('featurechange', function(wkt) {
-          instance.map.getTargetElement().parentElement.querySelector('[data-map-geometry-field]').value = wkt;
+          input.value = wkt;
         });
       });
+
+      // Run a handleInput() callback when input changes and attempt to
+      // import the new WKT.
+      input.oninput = handleInput;
+      function handleInput(e) {
```

Comment:
```suggestion
      input.addEventListener('input', (e) => {
```

See https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

![image](https://user-images.githubusercontent.com/30754460/236685652-418d4907-a9c5-4e9b-9d1b-02a2463b3d56.png)

Reply:
Thanks! I had a feeling there was a more "proper" way to do this. :-)

Fixup: https://github.com/farmOS/farmOS/pull/670/commits/9bdb4c39c535e87dc25d158faf3718047ff8eb55